docs/docs/building-services/gramine.md

```diff
@@ -74,21 +98,32 @@ You can specify the files' content in the MarbleRun manifest:
 ...
 ```
 
-Note that Gramine also allows to store files encrypted on the host's file system. The so called `protected files` require to initialize the protected files key by writing it hex-encoded to the virtual `protected_files_key` device:
+Note that Gramine also allows to store files encrypted on the host's file system. The so encrypted filesystem requires to initialize an encrypted files key by writing it to the virtual `protected_files_key` device.
+For debugging purposes, you can define the key directly in the Gramine manifest.
 
+```toml
+fs.insecure__keys.[KEY_NAME]] = "[16-byte hex value]"
```

Comment:
```suggestion
fs.insecure__keys.[KEY_NAME]] = "[32-character hex value]"
```
I know this existed before, but I would just call it the same way as Gramine does (32 character hex value).

Of course 32 chars hex string == 16-byte hex value but the latter can be slightly ambiguous if you switch between the two docs.